data/src/main/java/io/gnosis/data/repositories/ChainInfoRepository.kt

```diff
@@ -28,6 +28,8 @@ class ChainInfoRepository(
         }
     }
 
+    suspend fun update(chain: Chain) = chainDao.save(chain)
```

Comment:
initially there will be not chain in a db (as no safes are loaded). So first call of the update will save. Maybe calling it save would better represent it's function